### ISSUE TITLE
Your component is not compatible with Carthage

### ISSUE THREAD

**Issue by Dodig** [pronebird/UIScrollView-InfiniteScroll#23]
Hi
I'm using [Carthage](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) and when I tried to add your component in my Cartfile
```
github "pronebird/UIScrollView-InfiniteScroll" == 0.7.3
```
When I run the command `carthage update --platform iOS` I have this error:
```
*** Skipped building UIScrollView-InfiniteScroll due to the error:
Dependency "UIScrollView-InfiniteScroll" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/pronebird/UIScrollView-InfiniteScroll/issues/new
```

**Comment by TofPlay** [comment]
I have the same question

**Comment by pronebird** [comment]
As I don't use it myself, I'd have to find time to figure out how Carthage works. I would gladly accept a PR, I believe it would be faster if done by somebody who has experience with Carthage.

**Comment by pronebird** [comment]
Resolved with #26
